xiv_stats.php

```diff
@@ -419,7 +419,7 @@ function handleClass($row, $classDef, &$totalArray) {
     // Other mounts
     $literal_whale += in_array("Lunar Whale", $mounts) ? 1 : 0;
     $fmt_literal_whale = number_format($literal_whale);
-    $literal_whale_in_usd = $literal_whale*42;
```

Comment:
I think we just need to use `number_format($literal_whale*42)`. the 2nd parameter indicates the number of decimals so e.g. `$15,360,220.00` would be what a number would look like RN